gulp.config.js

```diff
@@ -81,12 +42,109 @@ module.exports = function() {
             // Seems to break sourcemaps, so conditionally minify css in gulpfile
             // outputStyle: 'compressed',
             precision: 8,
+            includePaths: [src + theme, src + base]
         },
         uglifySettings: {
             mangle: false,
         },
     };
 
+    setConfigPaths();
     return config;
+
+    //////////////////////////////////
+
+    // Fill the config with themes paths
+    function setConfigPaths() {
```

Comment:
Are we setting the theme paths here?  Or are we initializing the `Config` json object which will represent this theme?  Let's change the naming of the function to make that really clear.